packages/main-library/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "json-as-xlsx",
-  "version": "2.5.3",
+  "name": "json-as-xlsx-style",
```

Comment:
I think that changing this is going to cause some issues

Reply:
I apologize, I fixed it